OPCT-272: Create Sonobuoy mirror scripts to multi-arch images (#91)
redhat-openshift-ecosystem/provider-certification-tool: docs/dev.md

})
```

#### Sonobuoy image mirroring <a name="dev-image-mirror-sonobuoy"></a>

> The Sonobuoy images for Aggregator server and worker is mirrored to quay.io to prevent
> issues with docker hub network throtling.

Reviewer: ```suggestion
The Sonobuoy images for Aggregator server and worker are mirrored to quay.io to prevent
issues with docker hub network throttling.
```

The Sonobuoy image must follow the same of OPCT CLI (sonobuoy library defined in go.mod).

**Running the mirror**

> The mirror steps must be executed every time the Sonobuoy package is update on CLI, the following steps descripts how to start the mirror process:

Reviewer: ```suggestion
The mirror steps must be executed every time the Sonobuoy package is update on CLI, the following steps describes how to start the mirror process:
```

- Update the version `SONOBUOY_VERSION` in [`hack/image-mirror-sonobuoy/mirror.sh`][image-mirror-sonobuoy]
- Run mirror script
```bash
make image-mirror-sonobuoy
```
- Check the image in [quay.io/opct/sonobuoy](https://quay.io/repository/opct/sonobuoy?tab=tags)

**Running the mirror targetting custom repository**

```bash
SONOBUOY_VERSION=v0.56.11 MIRROR_REPO=quay.io/mrbraga/sonobuoy make image-mirror-sonobuoy
```

**Adding new Architecture**

> If you are looking to test new platform, you must add it when creating the manifest in [`hack/image-mirror-sonobuoy/mirror.sh`][image-mirror-sonobuoy]:

Reviewer: ```suggestion
If you are looking to test a new platform, you must add it when creating the manifest in [`hack/image-mirror-sonobuoy/mirror.sh`][image-mirror-sonobuoy]:
```
